app/controllers/bookings_controller.rb

```diff
@@ -7,14 +7,35 @@ def create
   end
 
   def show
+    @booking = Booking.find(params[:id])
   end
 
   def update
+    @booking = Booking.find(params[:id])
+    @booking.update(booking_params)
```

Comment:
In the create method, I have this check to make sure it's successful before redirect, otherwise I redirect to another page (In the update method, you might want to insert `render :edit`:

```
def create
    @booking = Booking.new(booking_params)
    @booking.user = current_user
    @booking.costume = @costume
    if @booking.save
      redirect_to bookings_path, notice: "Costume #{@costume.title} is booked sucessfully !"
    else
      # render :new
      redirect_to costume_path(@costume), alert: @booking.errors.full_messages.first
    end
  end 
```